xml_models/managers.py

```diff
@@ -2,6 +2,7 @@
 import xml_models.rest_client as rest_client
 from lxml import etree
 from xml_models.xpath_finder import MultipleNodesReturnedException
+from StringIO import StringIO
```

Comment:
This will fail in Python3 as the StringIO module has moved. See the way it is handled in the tests.